packages/components/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@labkey/components",
-  "version": "0.70.5",
+  "version": "0.70.0-fb-issue40734.6",
```

Comment:
Did you end up publishing any alpha package versions to Artifactory with these changes? The last decimal number here of `.6` would usually indicate that you are on the sixth alpha package version. I usually will just let the `yarn publish` command be the time when I set the alpha package version. In this case, for example, I would have chosen `0.70.5-fb-issue40734.0`

Reply:
Fixed